src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -369,6 +379,7 @@ class SQLPTRepresentationTest2 : public ::testing::Test {
   }
 
   virtual void TearDown() OVERRIDE {
+    chmod(kAppStorageFolder.c_str(), 755);
```

Comment:
@ZhdanovP why we are setting some access before removing? If we created this file, we should be able to remove it. If this file was created not by us, it should not be removed.

Reply:
@LuxoftAKutsan Hello, this test creates the folder with chmod - 00000 and tries to Init data base there, in this case, for delete this dir after test, whe need to change it permissions.

Reply:
it seems this is negative test case.